crates/edr_evm/src/miner.rs

```diff
@@ -44,7 +44,7 @@ pub struct MineBlockResultAndState<StateErrorT> {
 }
 
 /// The type of ordering to use when selecting blocks to mine.
-#[derive(Debug)]
+#[derive(Debug, Clone)]
```

Comment:
I find 37 results for `Debug, Clone` in the codebase. :) I agree that it'd be nice to have it consistent, but imo we should either have a lint for it or not bother about it.

Reply:
I don't know how we could write a lint for it? Have you done it before?

Reply:
`rustfmt` doesn't support it unfortunately, because there are some edge cases with third-party macros where automatic sorting doesn't play well, but writing a script that checks if derives are in a certain order should be simple enough and we could adjust the script if we run into a problem. In the meantime we could create a style guide and add it there to make the requirements clear?